[en] update cluster-scheduler-plugin (#4544)
DaoCloud/DaoCloud-docs: docs/en/docs/kpanda/user-guide/clusters/cluster-scheduler-plugin.md

This page describes how to deploy a second scheduler-plugins in a cluster.

## Why do we need scheduler-plugins?

The cluster created through the platform will install the native K8s scheduler-plugin, but the native scheduler-plugin 
has many limitations:

- The native scheduler-plugin cannot meet scheduling requirements, so you can use either 
  [CoScheduling](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/pkg/coscheduling),
  [CapacityScheduling](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/pkg/capacityscheduling)
  or other types of scheduler-plugins.
- In special scenarios, a new scheduler-plugin is needed to complete scheduling tasks without affecting the process of
  the native scheduler-plugin.
- Distinguish scheduler-plugins with different functionalities and achieve different scheduling scenarios by switching
  scheduler-plugin names.

This page takes the scenario of using the vgpu scheduler-plugin while combining the coscheduling plugin capability 
of scheduler-plugins as an example to introduce how to install and use scheduler-plugins.

## Installing scheduler-plugins

### Prerequisites

> - kubean is a new feature introduced in version v0.13.0, please ensure that the version is or above version v0.13.0.

Reviewer: ```suggestion
- kubean is a new feature introduced in v0.13.0, please ensure that your version is v0.13.0 or higher.
```

    Parameters:

    - `scheduler_plugins_enabled` Set to true to enable the scheduler-plugins capability.
    - You can enable or disable certain plugins by setting the `scheduler_plugins_enabled_plugins` or 
      `scheduler_plugins_disabled_plugins` options.

>     See [K8s Official Plugin Names](https://github.com/kubernetes-sigs/scheduler-plugins?tab=readme-ov-file#plugins) 

Reviewer: ```suggestion
      See [K8s Official Plugin Names](https://github.com/kubernetes-sigs/scheduler-plugins?tab=readme-ov-file#plugins) 
```
